package.json

```diff
@@ -193,6 +193,12 @@
                 "title": "Enable \"Snyk Advisor\"",
                 "description": "Discover the health (maintenance, community, popularity & security) status of your open source packages.",
                 "default": false
+              },
+              "lsIacScan": {
+                "type": "boolean",
+                "title": "(Preview) Enable Snyk Infrastructure as Code scans",
```

Comment:
Generally no need to specify preview here - you can leave it, since it doesn't matter anyway :)